conformance/lang/expressions/multiplicative-expr/division.balt

```diff
@@ -4033,3 +4033,851 @@ Description: Test integer division panic at module-level.
 Labels: multiplicative-expr, int:MIN_VALUE, DecimalNumber, unary-minus
 
 int a = int:MIN_VALUE / -1; // @panic integer overflow on division
+
+Test-Case: panic
+Description: Test 1 for decimal division panic on division by zero.
+Labels: multiplicative-expr, DecimalNumber, unary-minus, DecimalFloatingPointNumber, decimal, DecimalTypeSuffix
```

Comment:
Need to use `FloatingPointTypeSuffix` instead of `DecimalTypeSuffix`. Should fix in other places too.